build/Common.props

```diff
@@ -31,11 +31,12 @@
     <MicrosoftOwinPkgVer>[4.2.2,5.0)</MicrosoftOwinPkgVer>
     <MicrosoftPublicApiAnalyzersPkgVer>[3.3.3]</MicrosoftPublicApiAnalyzersPkgVer>
     <MicrosoftSourceLinkGitHubPkgVer>[1.1.1,2.0)</MicrosoftSourceLinkGitHubPkgVer>
-    <OpenTelemetryCoreLatestVersion>[1.3.2,2.0)</OpenTelemetryCoreLatestVersion>
+    <OpenTelemetryCoreLatestVersion>[1.4.0,2.0)</OpenTelemetryCoreLatestVersion>
     <OpenTelemetryCoreLatestPrereleaseVersion>[1.4.0-rc.4,2.0)</OpenTelemetryCoreLatestPrereleaseVersion>
```

Comment:
Please change all `OpenTelemetryCoreLatestPrereleaseVersion` to `OpenTelemetryCoreLatestVersion`

Or at least change this line to 1.4.0 instead of RC

Reply:
I was thinking about this. It sounds weird to have `OpenTelemetryCoreLatestPrereleaseVersion` point to a non-prerelease version 😀

We could change it 1.4.0 for now and then update it to 1.5.0-alpha.1 when it's out.

@cijothomas Thoughts?

Reply:
> We could change it 1.4.0 for now and then update it to 1.5.0-alpha.1 when it's out.

This sounds good.